src/helpers/tags.ts

```diff
@@ -0,0 +1,19 @@
+// Context
+export const PARENT_SPAN_ID = 'x-datadog-parent-id'
+export const TRACE_ID = 'x-datadog-trace-id'
+
+export const CI_ENV_PARENT_SPAN_ID = 'X_DATADOG_PARENT_ID'
+export const CI_ENV_TRACE_ID = 'X_DATADOG_TRACE_ID'
+
+// Build
+export const CI_PIPELINE_URL = 'ci.pipeline.url'
+export const CI_PROVIDER_NAME = 'ci.provider.name'
+
+// Git
+export const GIT_BRANCH = 'git.branch'
+export const GIT_SHA = 'git.commit.sha'
+
+// Other
+export const INSTRUCTION = 'instruction'
+export const EXIT_CODE = 'exit_code'
+export const ERROR = 'error'
```

Comment:
Not a big deal but I'm not sure those constants have their place in `helpers`, it looks like they are specific to the `trace` command?